packages/odie-client/src/constants.ts

```diff
@@ -49,76 +49,59 @@ export const ODIE_FORWARD_TO_ZENDESK_MESSAGE = __(
 	__i18n_text_domain__
 );
 
-export const ODIE_TRANSFER_MESSAGE = (
-	shouldUseHelpCenterExperience: boolean | undefined
-): Message => ( {
-	content: shouldUseHelpCenterExperience
-		? __( "Help's on the way!", __i18n_text_domain__ )
-		: __( "We're connecting you to our support team.", __i18n_text_domain__ ),
-	role: 'bot',
-	type: 'message',
-	context: {
-		flags: {
-			hide_disclaimer_content: true,
-			show_contact_support_msg: true,
+export const ODIE_TRANSFER_MESSAGE: Message[] = [
+	{
+		content: __( "We're connecting you to our support team.", __i18n_text_domain__ ),
+		role: 'bot',
+		type: 'message',
+		context: {
+			flags: {
+				hide_disclaimer_content: true,
+				show_contact_support_msg: true,
+			},
+			site_id: null,
 		},
-		site_id: null,
 	},
-} );
+];
 
-export const ODIE_TRANSFER_MESSAGE_NEW = (
-	shouldUseHelpCenterExperience: boolean | undefined
-): Message[] => {
-	let transferMessage: Message[] = [];
-	if ( shouldUseHelpCenterExperience ) {
-		transferMessage = [
-			{
-				content: __( "We're connecting you to our support team.", __i18n_text_domain__ ),
-				role: 'bot',
-				type: 'message',
-				context: {
-					flags: {
-						hide_disclaimer_content: true,
-						show_contact_support_msg: false,
-						show_ai_avatar: false,
-					},
-					site_id: null,
-				},
+export const ODIE_TRANSFER_MESSAGE_NEW: Message[] = [
+	{
+		content: __( "We're connecting you to our support team.", __i18n_text_domain__ ),
```

Comment:
Mmm before it was like

```
content: shouldUseHelpCenterExperience
		? __( "Help's on the way!", __i18n_text_domain__ )
```

I believe this should be the correct text.

Sorry again, the screenshot maybe was misleading. The copy is in the doc 1ptULS7zslhT8vPZEL3hD5CMwv2hALByQB36hMhzQwKI-gdoc

To make this easier, copy as it was before is correct

Reply:
```
New copy: Help’s on the way! 
Feel free to share more details while we connect you to support. 
```

Reply:
Thanks for the updated copy. I updated my PR to address this.